public/app/features/users/UsersActionBar.tsx

```diff
@@ -52,6 +53,13 @@ export const UsersActionBarUnconnected = ({
   // 2) new basic auth users can be created for this instance (!config.disableLoginForm).
   const showInviteButton: boolean = canAddToOrg && !(config.disableLoginForm && config.externalUserMngInfo);
 
+  const onExternalUserMngClick = () => {
+    reportInteraction('users_admin_actions_clicked', {
+      category: 'org_users',
+      item: 'manage_users_grafanacom',
```

Comment:
nit `grafanacom` is part of the configured value, maybe it worth to change it by something more generic like `manage_users_external`.